### DISCUSSION TITLE
Percentage of Billed Charges

### DISCUSSION THREAD

**Discussion by angelamarie0975** [CMSgov/price-transparency-guide#315] [General]
@shaselton-usds Has there been any more discussions on the % of Billed Charges topic at CMS? Since the billed charge is not known at the provider contract level - we just have the contract states just a % rate, what is the expectation of the insurers to get the billed charge? There are discussions about different approaches being used to determine the billed amount, but I don't know if there has been anything defined as to how we should be deriving the billed charge. Is there any guidance for this from CMS?

In the discussion from https://github.com/CMSgov/price-transparency-guide/discussions/23 - @Ginger228 had made a suggestion to add 'percent_charged' as a negotiated type and then have the negotiated rate be the percentage amount. I like this idea instead of using historical claim data as others have commented about using to get to the billed amount. 

If we need to use historical claims data to get the billed charge, then I have some questions/concerns for using that data:

1. What timeframe would we use to gather the current billed charge per provider? For the OON file - we have the 180 day look back for the historical claims data, but this would not necessarily be the most current amount that the providers are charging.
2.  What happens when we don't have any current data for those procedure/provider combinations? How would we validate what the billed amount is in this scenario if we are required to use the claims historical data?
3. When we get the data back - would we do an average on the billed amount if there are multiple billed charges during the same timeframe that we pull or would we pull the most recent charge?
4. If we use claims data, then would we need to follow the same 20 claim rule as OON file? If so, this may prevent us from being able to get a billed amount for ALL procedure codes. See question 2. 

@taylorpatriciab, @BobSyracuse, @ryan-thomas-peterson, @tdfow - have any of you made any headway on this topic? What are your thoughts?

**Comment by tdfow** [comment]
@angelamarie0975 I have asked the question in the prior webinar sessions and have been told it will be discussed in the next webinar, so keep hoping it will be addressed soon.   I feel there  needs to be something to identify either the basis of the charge and/or the percentage of charge value.  We do not have a charge amount for identifying the rate for these scenarios, so currently using $1 as the basis of charge since the schema requires the negotiated rate be published in a dollar value.   For percentage of charge, I previously suggested adding a value to the negotiated type, so at least the rate had better interpretation for being associated with percentage of charge.  If there was a new value for 'definition' of the rate and then defined basis of charge amount then at least the rate could be interpreted correctly.  Hopefully, CMS will address in the next webinar. :)

**Reply by angelamarie0975** [reply]
Here's hoping this would get addressed along with some other items. :)

**Comment by shaselton-usds** [comment]
@angelamarie0975 Please checkout this [PR](https://github.com/CMSgov/price-transparency-guide/pull/393) to see if it addresses your concerns. Thanks for the patience.

**Reply by angelamarie0975** [reply]
It did address my concerns @shaselton-usds! That was what we were hoping to hear! Thanks so much!!
